Extension/src/common.ts

```diff
@@ -776,6 +777,17 @@ export function promptReloadWindow(message: string): void {
     });
 }
 
+export async function CreateTempFileWithPostfix(postfix: string): Promise<tmp.FileResult> {
+    return new Promise<tmp.FileResult>((resolve, reject) => {
+        tmp.file({ postfix: postfix }, (err, path, fd, cleanupCallback) => {
+            if (err) {
+                return reject();
```

Comment:
`reject` will result in a `throw` when awaited.  I think you should call `reject(err)` to pass the `Error` along.